lib/matplotlib/ticker.py

```diff
@@ -1231,6 +1231,15 @@ def __init__(self, unit="", places=None, sep=" "):
         self.places = places
         self.sep = sep
 
+    def fix_minus(self, s):
```

Comment:
you can just do `fix_minus = ScalarFormatter.fix_minus` to not duplicate the code.

Reply:
fixed.